### PR TITLE
Ref/cordova wkwebview 01

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

File: app-template/config-template.xml [hunk 1]
```diff
@@ -51,14 +47,17 @@
...
-    <plugin name="cordova-plugin-qrscanner" spec="2.5.0" />
+    <plugin name="cordova-plugin-qrscanner" spec="2.6.0" />
+    <plugin name="cordova-plugin-add-swift-support" spec="1.7.2" />
     <plugin name="cordova-plugin-splashscreen" spec="5.0.2" />
     <plugin name="cordova-plugin-statusbar" spec="2.4.2" />
-    <plugin name="cordova-plugin-inappbrowser" spec="2.0.2" />
-    <plugin name="cordova-plugin-touch-id" spec="3.3.1" />
+    <plugin name="cordova-plugin-inappbrowser" spec="3.0.0" />
+    <plugin name="cordova-plugin-touch-id" spec="3.4.0">
+      <variable name="FACEID_USAGE_DESCRIPTION" value=" " />
```
Comment (matiu) [inline app-template/config-template.xml]: should we add an id here?

File: app-template/config-template.xml [hunk 2]
```diff
@@ -51,14 +47,17 @@
     <plugin name="cordova-plugin-android-fingerprint-auth" spec="1.4.3" />
     <plugin name="cordova-plugin-device" spec="2.0.1" />
     <plugin name="cordova-plugin-globalization" spec="1.0.9" />
-    <plugin name="cordova-plugin-qrscanner" spec="2.5.0" />
+    <plugin name="cordova-plugin-qrscanner" spec="2.6.0" />
+    <plugin name="cordova-plugin-add-swift-support" spec="1.7.2" />
```
Comment (matiu) [inline app-template/config-template.xml]: why we need this?

File: app-template/config-template.xml [hunk 3]
```diff
@@ -51,14 +47,17 @@
...
-    <plugin name="cordova-plugin-inappbrowser" spec="2.0.2" />
-    <plugin name="cordova-plugin-touch-id" spec="3.3.1" />
+    <plugin name="cordova-plugin-inappbrowser" spec="3.0.0" />
+    <plugin name="cordova-plugin-touch-id" spec="3.4.0">
+      <variable name="FACEID_USAGE_DESCRIPTION" value=" " />
+    </plugin>
     <plugin name="cordova-plugin-whitelist" spec="1.3.3" />
-    <plugin name="cordova-plugin-x-socialsharing" spec="5.4.1" />
-    <plugin name="cordova-plugin-x-toast" spec="2.6.0" />
+    <plugin name="cordova-plugin-x-socialsharing" spec="5.4.3" />
```
Comment (matiu) [inline app-template/config-template.xml]: lets not upgrade plugin ins unless there is a security issue.

x-socialsharing // x-toast // qr-scanner

File: app-template/config-template.xml [hunk 4]
```diff
@@ -119,6 +116,14 @@
     </platform>
 
     <platform name="android">
+      <access origin="https://bws.bitpay.com" />
```
Comment (matiu) [inline app-template/config-template.xml]: do we need this even we use CSP?

File: app-template/config-template.xml (hunk 3 above)
Comment (matiu) [reply app-template/config-template.xml]: + inappbrowser

File: app-template/config-template.xml (hunk 1 above)
Comment (cmgustavo) [reply app-template/config-template.xml]: No, the value is " " (empty). Description for FaceId is optional for `NSFaceIDUsageDescription` in plist file.

File: app-template/config-template.xml (hunk 2 above)
Comment (cmgustavo) [reply app-template/config-template.xml]: `cordova-plugin-qrscanner` requires `cordova-plugin-add-swift-support`, but it 's not fixed version. Check this line:
```

Comment (cmgustavo) [reply app-template/config-template.xml]: Install a fixed version of `cordova-plugin-add-swift-support` in our side, would avoid to install any update of this.

Comment (matiu) [reply app-template/config-template.xml]: ack

File: app-template/config-template.xml (hunk 1 above)
Comment (matiu) [reply app-template/config-template.xml]: ack

File: app-template/config-template.xml (hunk 4 above)
Comment (matiu) [reply app-template/config-template.xml]: ack

File: app-template/config-template.xml (hunk 3 above)
Comment (matiu) [reply app-template/config-template.xml]: ok

Comment (cmgustavo) [reply app-template/config-template.xml]: qr-scanner is ok. Version number on package.json was 2.6.0 for long time (maybe we forget to change it on config.xml). Previous version may break the building process with cordova 8: https://github.com/bitpay/cordova-plugin-qrscanner/issues/144#issuecomment-389976578

**Pull request merged**